src/base/bittorrent/tracker.h

```diff
@@ -49,30 +49,11 @@ namespace Http
 
 namespace BitTorrent
 {
-    struct Peer
-    {
-        QString ip;
-        QString peerId;
-        int port;
-
-        bool operator!=(const Peer &other) const;
-        bool operator==(const Peer &other) const;
-        QString uid() const;
-        libtorrent::entry toEntry(bool noPeerId) const;
-    };
-
-    struct TrackerAnnounceRequest
-    {
-        QString infoHash;
-        QString event;
-        int numwant;
-        Peer peer;
-        // Extensions
-        bool noPeerId;
-    };
+    struct Peer;
```

Comment:
I told you to move these declarations outside the BitTorrent namespace. No need to put these **private** name here.